File: ExcelDivider/src/main/java/ru/tsystems/divider/dao/api/TaskDao.java
package ru.tsystems.divider.dao.api;

import ru.tsystems.divider.entity.Task;

public interface TaskDao extends GeneralDao<Task> {

    /**
     * Finding task by key
     *
     * @param key
     *            Key
     * @return {@link Task}
     */
    Task getBykey(String key);
}

File: ExcelDivider/src/main/kotlin/ru/tsystems/divider/dao/api/TaskDao.kt
package ru.tsystems.divider.dao.api

import ru.tsystems.divider.entity.Task

interface TaskDao : GeneralDao<Task> {

    /**
     * Finding task by key
     *
     * @param key
     * Key
     * @return [Task]
     */
    fun getBykey(key: String): Task?
}
